doc/releases/changelog-dev.md

```diff
@@ -4,92 +4,127 @@
 
 <h3>New features since last release</h3>
 
+<h4>Easy to import circuits 💾</h4>
```

Comment:
Was this discussed last we spoke about the new features? 🤔

Reply:
No sorry - this is a new one coming in where it will be easier to import circuits from Qiskit.